src/main/java/com/newts/newtapp/api/application/InputBoundary.java

```diff
@@ -13,5 +14,5 @@
      * Accepts a request.
      * @param request   a request stored as a RequestModel
      */
-    ReturnType request(RequestModel request) throws ExceptionType;
+    ReturnType request(RequestModel request) throws ExceptionType, UserNotFoundInConversation;
```

Comment:
I think the old code is fine. Try deleting UserNotFoundInConversation at the end.